tests/test_barcodes.py

```diff
@@ -17,6 +17,22 @@ def test_get_new_barcode(client):
     assert response.status_code == HTTPStatus.CREATED
 
 
+def test_get_new_barcodes_group_as_url_param(client):
+    response = client.post("/barcodes_group/SANG/new?count=3")
+    assert response.json == {
+        "barcodes_group": {"barcodes": ["SANG-30D404", "SANG-30D413", "SANG-30D422"], "id": 1}
+    }
+    assert response.status_code == HTTPStatus.CREATED
+
+
+def test_get_new_barcodes_group_as_json_param(client):
```

Comment:
Add a test without providing the count and see what it raises?

Reply:
Ok, added tests